zos_concepts/gdg_datasets/create_copy_submit/UPTIME

```diff
@@ -0,0 +1,21 @@
+//******************************************************************************
```

Comment:
Can you add a copyright header to this file - see [here](https://github.com/IBM/z_ansible_collections_samples/blob/main/zos_concepts/jobs/submit_query_retrieve/files/HELLO.jcl) for an example